index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/styles.css">
+    <title>Roland Garros 2023</title>
+</head>
+<body>
+    <header>
+      <ul class="lato">
+        <li>
+          <a href="#">
+            <i class="bi bi-facebook"></i>
+          </a>
+        </li>
+          <li>
+            <a href="#">
+              <i class="bi bi-twitter"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#">English</a>
+          </li>
+          <li>
+            <a href="#">My Page</a>
+          </li>
+          <li>
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+        <i class="bi bi-x close-menu"></i>
+        <nav>
+          <a href="./index.html"><img src="./assets/images/logo-rg.svg" class="logo" alt="Roland Garros"></a>
+          <ul class="lato">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Tournament</a></li>
+            <li><a href="#">Tickets</a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#" class="highlighted-menu">Roland Garros 2023</a></li>
+          </ul>
+        </nav>
+    </header>
+    <section id="headline">
+      <i class="bi bi-list" id="menu-button"></i>
+      <h3 class="lato">"Hello! Tennis Fans"</h3>
+      <h1 class="cocogoose">ROLAND GARROS 2023 <br>
+        THE MAIN TENNIS EVENT IN THE SEASON IS COMING
+        IN MAY 2023
+      </h1>
+      <p class="lato">Be part of it all in Opening Week (22-26 May)<br>
+      Come and watch the qualifying matches, the stars on the practice courts and much more besides in Opening Week at this year’s Parisian Grand Slam event.</p>
+      <h2 class="cocogoose">22 MAY - 11 JUN 2023</h2>
+      <h4 class="lato">©2023 - all rights reserved
+        Fédération Française de Tennis</h4>
+    </section>
```

Comment:
- Kindly make sure that you add the appropriate  background images to this section to make it look luck the design provided as required in the project :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/235262815-cfd6211a-6926-44c4-9b51-92d700e25116.png)

## The design provided 


![image](https://user-images.githubusercontent.com/95297251/235262905-8943cc79-fb68-4331-b084-5a939f81c294.png)

Reply:
![issue-02](https://user-images.githubusercontent.com/86327867/235285609-c967226d-2ea5-4136-82eb-ea4a6338ee14.jpg)

Fixed the background image in index.html